src/tools/interopserver/InteropServer.h

```diff
@@ -43,13 +43,24 @@ const QUIC_BUFFER QuackAckBuffer = { sizeof("quack-ack") - 1, (uint8_t*)"quack-a
 //
 // Exits if there is a failure.
 //
+#ifdef QUIC_BUILD_STATIC
 #define EXIT_ON_FAILURE(x) do { \
     auto _Status = x; \
     if (QUIC_FAILED(_Status)) { \
        printf("%s:%d %s failed!\n", __FILE__, __LINE__, #x); \
+       MsQuicUnload(); \
```

Comment:
Since this is not going to also cause an MsQuicClose, this is going to assert, as the library is still open. We might need to use some kind of AtExit handler here, or figure out another way to safely exit the app.